crates/pallet-erc20-support/src/lib.rs

```diff
@@ -65,6 +70,17 @@ pub mod pallet {
         /// The currency to be exposed as ERC20 token.
         type Currency: Currency<AccountIdOf<Self, I>>;
 
+        /// Allowance balance type.
+        type AllowanceBalance: From<BalanceOf<Self, I>>
+            + Sub<Output = Self::AllowanceBalance>
+            + PartialOrd
+            + Default
+            + Copy
+            + Codec
+            + EncodeLike
+            + MaxEncodedLen
+            + scale_info::TypeInfo;
```

Comment:
Some of these can be replaced with `Parameter + Member`

Reply:
https://github.com/humanode-network/humanode/pull/809/commits/635dfa44fc93abf585cfb822911839161d65a2d8